poc/pkg/values.yaml

```diff
@@ -85,7 +85,6 @@ affinity: {}
 #TODO jonachang: need to add a secret in other PR
 istio:
   name: aks-istio-ingressgateway-external
-  namespace: aro-rp-poc
```

Comment:
Since we specify namespace during helm install, we don't need to include this here